docs/tutorial-01.md

```diff
@@ -0,0 +1,210 @@
+
+    dnf install --installroot $scratchmnt --release 26 bash coreutils --setopt install_weak_deps=false -y
+
+Let's try it out:
+
+    # buildah run $newcontainer bash
+    bash-4.4# cd /usr/bin
+    bash-4.4# exit
+
+Notice we have a `/usr/bin` directoy in the newcontianers image layer. Let's first copy a simple file. Create a file called runecho.sh contains the following:
+
+    #!/bin/bash
+    for i in `seq 0 9`;
+    do
+    	echo "This is a new container from ipbabble [" $i "]"
+    done
+
+Change the permissions on the file so that it can be run:
+
+    chmod +x runecho.sh
+    
+With `buildah` files can be copied into the new image and we can also configure the image to run commands. Let's copy this new command into the container's `/usr/bin` directory and configure the container to run the command when the container is run: 
+
+    buildah copy $newcontainer ./runecho.sh /usr/bin
+    buildah config --cmd /usr/bin/runecho.sh $newcontainer
+    
+Now run the container:
+
+    # buildah run $newcontainer
+    This is a new container from ipbabble [ 0 ]
+    This is a new container from ipbabble [ 1 ]
+    This is a new container from ipbabble [ 2 ]
+    This is a new container from ipbabble [ 3 ]
+    This is a new container from ipbabble [ 4 ]
+    This is a new container from ipbabble [ 5 ]
+    This is a new container from ipbabble [ 6 ]
+    This is a new container from ipbabble [ 7 ]
+    This is a new container from ipbabble [ 8 ]
+    This is a new container from ipbabble [ 9 ]
+
+It works! Congratulations, you have built a new OCI container from scratch that uses bash scripting. Let's add some more configuration information.
+
+    buildah config --created-by "ipbabble"  $newcontainer
+    buildah config --author "wgh at redhat.com @ipbabble" --label fedora26-bashecho:latest $newcontainer
+ 
+We can inspect the container's metadata using the `inspect` command:
+ 
+    buildah inspect $newcontainer
+
+We should probably unmount and commit the image:
+
+     buildah unmount $newcontainer
+     buildah commit $newcontainer fedora-bashecho
+     buildah images
+     
+And you can see there is a new image called `fedora-bashecho:latest`. You can inspect the new image using:
+
+    buildah inspect --type=image fedora-bashecho
+
+## OCI images built using buildah are portable
+
+Let's test if this new OCI image is really portable to another OCI technology like docker. First you shouuld install docker and start it.
+
+    dnf -y install docker
+    systemctl start docker
+    
+The we need to push the image from the `containers/image` across to docker's storage area which is under `/var/lib/docker`. This is managed by the docker dameon. We can do this telling buildah to push to the docker protocol called docker-daemon.
```

Comment:
typo: daemon

Reply:
Maybe "Let's copy that image from where and how containers/storage stores it to where and how the docker daemon stores its images, so that we can run it using docker."?

Reply:
Going with:
"Let's copy that image from where and how containers/storage stores it to where and how the docker daemon stores its images, so that we can run it using docker. We can achieve this using `buildah push`. This copies the image to docker's repository area which is located under `/var/lib/docker`. Docker's repository is managed by the docker daemon. This needs to be explicitly stated by telling buildah to push to the docker repository protocol using `docker-daemon:`."